Sources/DreamsViewController.swift

```diff
@@ -25,18 +25,22 @@ public class DreamsView: UIView {
     internal var webService: WebServiceType = WebService()
```

Comment:
I think we should try to inject dependencies. I understand this may be outside of this PR scope but wanted to point that out anyway.

Reply:
👍 Yes, will be treated in a separate PR.